docker-compose.circleci.yml

```diff
@@ -2,9 +2,9 @@
 version: '3.7'
 services:
   db_migrate:
-    image: "${AWS_ACCOUNT_ID}.dkr.ecr.us-west-2.amazonaws.com/easi-db-migrate:${CIRCLE_SHA1}"
+    image: '${AWS_ACCOUNT_ID}.dkr.ecr.us-west-2.amazonaws.com/easi-db-migrate:${CIRCLE_SHA1}'
   easi:
-    image: "${AWS_ACCOUNT_ID}.dkr.ecr.us-west-2.amazonaws.com/easi-backend:${CIRCLE_SHA1}"
+    image: '${AWS_ACCOUNT_ID}.dkr.ecr.us-west-2.amazonaws.com/easi-backend:${CIRCLE_SHA1}'
     environment:
       - APPLICATION_VERSION
       - APPLICATION_DATETIME
```

Comment:
The `easi` service in docker-compose.circleci.yml needs a reference to the `FLAG_SOURCE` env var. Try adding `- FLAG_SOURCE` to pick up the value set in the CircleCI config file under the `integration_tests` environment section. If that doesn't work, try setting the env var directly (``- FLAG_SOURCE=LOCAL`).